### PR TITLE
(#1815) SummaryBox Header Font Size

### LINKED ISSUE #1815: CR: Change Font Size on Summary Box
### Request Description
The current NCIDS Summary Box uses h2 styling (token size LG) for its heading, which is oversized for the component type and usage. Therefore, this request is to reduce the size of the heading to token size MD (font size 20px/Line-height-2), while keeping the heading as an h2 for the page hierarchy.

### What's the expected change?
- Change the font size of the NCIDS Summary Box Heading to be token size MD (font size 20px/line-height-2) while keeping it as an h2 for the page hierarchy.

### What's the current functionality?
- The font size of the NCIDS Summary Box Heading is token size LG.

### Resources
- [Comp](https://www.figma.com/design/GCOniv8OTyj9OC1udKwo6S/Mini-Landing-Pages?node-id=3919-18314&t=CwQZuwWUEKmoJnpt-0)

### What's the updated acceptance criteria?
```gherkin
Scenario: A site visitor views an NCIDS Summary Box on a webpage
  Given a site visitor is on a webpage
    And the webpage has an NCIDS Summary Box
  When the visitor views the NCIDS Summary Box
    And the NCIDS Summary Box has a heading
  Then the heading will be token size MD (font size 20px/line-height-2)
    And the heading will be an h2 in the page hierarchy
```

### Related Tickets
* Digital Platform Ticket: [#4389](https://github.com/NCIOCPL/cgov-digital-platform/issues/4389)

### PR DESCRIPTION
**Pull request by karlikpj**
Closes #1815

## Pull Request Details

_Add description_

Closes #<TICKET-NUMBER>

## Author PR Checklist
Items that the author of the PR is responsible for checking before submitted the PR.

### General:
- [x] I have reviewed the acceptance criteria defined in the ticket and ensured the work has been completed.
- [x] The commit message passes [all quality commit message standards](https://github.com/NCIOCPL/ncids/wiki/Creating-Quality-Commit-Messages).
- [x] Unit tests have been updated or created to reflect any javascript changes.
- [x] Storybook scenarios have been updated or created to reflect any html/css/js changes.

### Accessibility:
- [x] WCAG 2.1 Level AA requirements have been met.

### Development:
- [x] Any new or updated javascript code has 100% unit test coverage.
- [x] New or updated breakpoints have regression images.
- [x] Breaking changes have been thoroughly documented in the PR.

## Product Reviewer PR Checklist
Items the product team is responsible for reviewing.

### General:
- [x] There are no unexpected or unapproved regression image changes. 
- [x] Functionality of interactive elements meet the acceptance criteria.
- [x] The product is visually and functionally the same across the different browsers.

### Accessibility:
- [x] AxeDev Tools: there are no new or outstanding accessibility issues introduced in this PR.
- [x] Lighthouse: scores have not noticeably decreased during this PR.
- [x] Wave: there are no new errors or contrast errors introduced in this PR.

## Design Reviewer PR Checklist
Items the design team is responsible for reviewing.  

### General:
- [ ] New or updated features introduced in this PR are developed mobile-first.
- [ ] Breakpoint changes and regression images match those breakpoints.
- [ ] This PR has been tested in all supported browsers at all breakpoints.

## Developer Reviewer PR Checklist
Items the development team is responsible for reviewing.

### General:
- [ ] New code passes code quality standards set by industry standards.
- [ ] The expected Storybook stories have been added or updated for the new or updated feature.
- [ ] The expected unit tests have been added or updated for the new or updated feature.

### Accessibility:
- [ ] VoiceOver: Described content matches with what was expected.
- [ ] Keyboard navigation: new or updated features and content are navigable via the keyboard.

### CODE REVIEW

File: pnpm-lock.yaml
Comment (olitharp-nci) [inline pnpm-lock.yaml:1]: you should have no pnpm-lock changes

File: testing/ncids-css-testing/.backstop/reference/ncids_nci-autocomplete_default_match-disabled_below-min_0_document_0_mobile.png
Comment (olitharp-nci) [inline testing/ncids-css-testing/.backstop/reference/ncids_nci-autocomplete_default_match-disabled_below-min_0_document_0_mobile.png:1]: remove this

**Review by olitharp-nci — CHANGES_REQUESTED** [review]
Make sure you review your own PR changes to confirm nothing unexpected was committed. Clean these up and make sure the ticket is up to date with develop.

**Review by olitharp-nci — APPROVED** [review]

File: pnpm-lock.yaml
Comment (karlikpj) [reply pnpm-lock.yaml:1]: Patch Doc says lock will change - I didnt touch anything else but the patch here?

**Pull request merged**